app/templates/events/view/tickets/discount-codes.hbs

```diff
@@ -0,0 +1,93 @@
+<div class="ui stackable grid">
+  <div class="row">
+    <div class="eight wide column">
+      <h2 class="ui header">
+        {{t 'Discount Codes'}}
+      </h2>
+    </div>
+    <div class="eight wide {{unless device.isMobile 'right aligned'}} column">
+      <button class="ui teal button">
+        <i class="plus icon"></i>
+        {{t 'Create a discount'}}
+      </button>
+    </div>
+  </div>
+  <div class="row">
+    <div class="ui hidden divider"></div>
+    <div class="ui {{if device.isMobile 'vertical'}} basic buttons">
+      <div class="ui button active">
+        {{t 'All'}}
+      </div>
+      <div class="ui button">
+        {{t 'Active'}}
+      </div>
+      <div class="ui button">
+        {{t 'Inactive'}}
+      </div>
+    </div>
+    <table class="ui very basic table">
+      <thead>
+        <tr>
+          <th>
+            #
+          </th>
+          <th>
+            {{t 'Discount code'}}
+          </th>
+          <th>
+            {{t 'Discount code URL'}}
+          </th>
+          <th>
+            {{t 'Discount Per Ticket'}}
+          </th>
+          <th>
+            {{t 'Validity'}}
+          </th>
+          <th>
+            {{t 'Status'}}
+          </th>
+          <th>
+            {{t 'Action'}}
+          </th>
+        </tr>
+      </thead>
+      <tbody>
+        {{#each model as |discount|}}
+          <tr>
+            <td>
+              {{discount.id}}
+            </td>
+            <td>
+              {{discount.code}}
+            </td>
+            <td>
+              <a>{{discount.url}}</a>
+            </td>
+            <td>
+              US $ {{discount.amount}}
+            </td>
+            <td>
+              {{discount.validity}}
+            </td>
+            <td>
+              <a class="ui green label">{{discount.status}}</a>
+            </td>
+            <td class="centre aligned">
+              <div class="ui compact basic buttons">
```

Comment:
These buttons are overflowing the width of the table. 
![screenshot from 2017-06-07 19-19-30](https://user-images.githubusercontent.com/12807846/26881849-4d6218fc-4bb6-11e7-9978-318626a4a314.png)